cinema/admin.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
 
-from .models import (
+from cinema.models import (
+    Actor,
     CinemaHall,
     Genre,
     Actor,
```

Comment:
The `Actor` model is imported twice. You should remove the duplicate import to clean up the code.